statusHandler/statusMetricsProvider.go

```diff
@@ -245,6 +245,49 @@ func (sm *statusMetrics) NetworkMetrics() map[string]interface{} {
 	return networkMetrics
 }
 
+// RatingsMetrics will return metrics related to current configuration
+func (sm *statusMetrics) RatingsMetrics() map[string]interface{} {
```

Comment:
please add an unit test for this. it's hard to understand the desired output after the for range

Reply:
will add in the following PR :+1: